pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,64 @@
 # Image classification
-
-Models that are able to classify images.
+## Description of the models
+### Performance results pivot table
 
 | Model Name | Complexity (GFLOPs) | Size (Mp) | Top-1 on ImageNet (%) | Links |
 | ---        | ---                 | ---       | ---                   | ---                   |
 | MobilenetV2 1.0 | 0.66           | 3.50      | 73.0                  | [model template](./mobilenet_v2_w1/template.yaml)
 
-## Datasets
-
-The model was initially trained on [ImageNet](http://image-net.org/challenges/LSVRC/2012/), but can be trained from scratch or fine tuned to classify arbitrary images.
-
+| Model Name | Complexity (GFLOPs) | Size (Mp) | Latency (ms) | FPS | AVG mAP (%) | AVG Top-1 (%) | AVG Top-5 (%) | Links |
```

Comment:
Did you remove MobileNetV2? if yes please remove corresponding table.

Reply:
Sorry for that. It turns out, that README.md hasn't been updated for some reasons and a few stupid mistakes were pushed to the branch. Fixed that.